client-src/elements/form-definition.ts

```diff
@@ -1,103 +1,41 @@
 import {html, TemplateResult} from 'lit';
 import * as enums from './form-field-enums';
-
-interface BaseBrowser {
-  view: {
-    val: string;
-    text: string;
-    url: string;
-    notes: string;
-  };
-}
-
-interface ChromeBrowser extends BaseBrowser {
-  bug: string;
-  blink_components: string[];
-  devrel: string[];
-  owners: string[];
-  prefixed: boolean;
-  status: {val: string};
-  desktop: string;
-  android: string;
-  webview: string;
-  ios: string;
-}
-
-interface FirefoxBrowser extends BaseBrowser {}
-interface SafariBrowser extends BaseBrowser {}
-interface WebDevBrowser extends BaseBrowser {}
-interface OtherBrowser {
-  view: {notes: string};
-}
-
-//resources and standards used in Feature object
-interface Resources {
-  docs: string[];
-  samples: string[];
-}
-
-interface Standards {
-  spec: string;
-  maturity: {val: number; text: string};
-}
-
-type FeatureName = string;
-
-interface Feature {
-  name?: FeatureName;
-  category_int: number;
-  enterprise_feature_categories: string[];
-  feature_type_int: number;
-  intent_stage_int: number;
-  standards: Standards;
-  tag_review_status_int: number;
-  security_review_status_int: number;
-  privacy_review_status_int: number;
-  resources: Resources;
-  tags: string[];
-  browsers: {
-    chrome: ChromeBrowser;
-    ff: FirefoxBrowser;
-    safari: SafariBrowser;
-    webdev: WebDevBrowser;
-    other: OtherBrowser;
-  };
-}
+import {Feature} from '../js-src/cs-client';
 
 interface FormattedFeature {
   category: number;
   enterprise_feature_categories: string[];
   feature_type: number;
   intent_stage: number;
   accurate_as_of: boolean;
-  spec_link: string;
+  spec_link: string | null;
```

Comment:
To the extent that these are actually optional properties, they'll become

```suggestion
  spec_link?: string;
```